src/bundle.rs

```diff
@@ -1,5 +1,6 @@
 //! Structs related to bundles of Orchard actions.
 
+pub mod burn_validation;
 use alloc::vec::Vec;
 
 pub mod commitments;
```

Comment:
_From str4d ([r2585521988](https://github.com/zcash/orchard/pull/471#discussion_r2585521988))_

Wow the imports in this file are scuffed for some reason. But no need to make them moreso by mixing module definitions with unrelated imports.
```suggestion
use alloc::vec::Vec;

pub mod burn_validation;
pub mod commitments;
```

Reply:
FIXED